### PR TITLE
Prompter: send message when getImage is called

### PR DESCRIPTION
**Pull request by bencodeorg**
Sends a message ("GET_IMAGE") to Javalab when getImage is called, and includes the prompt that should be displayed in the `details`.

## Testing story

Added unit test, and tested manually that I saw these messages appearing when I called Theater.prompter.getImage("") from Javalab.

### CODE REVIEW

File: org-code-javabuilder/playground/build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ java {
 
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
-    // https://mvnrepository.com/artifact/org.mockito/mockito-core
+    // https://mvnrepository.com/artifact/org.mockito/mockito-inline
```
Comment (sanchitmalhotra126) [inline org-code-javabuilder/playground/build.gradle:19]: Thanks for updating these 👍

File: org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+import org.code.protocol.GlobalProtocol;
+import org.code.protocol.OutputAdapter;
 
 public class Prompter {
-  protected Prompter() {}
+  private final OutputAdapter outputAdapter;
+
+  protected Prompter() {
+    this.outputAdapter = GlobalProtocol.getInstance().getOutputAdapter();
+  }
```
Comment (sanchitmalhotra126) [inline org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java:13]: nit: might want to have a second constructor that takes in `outputAdapter` as a parameter and call that here, for testing purposes (similar to `Board` and `Stage`) so you don't have to mock the global protocol instance

File: org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
 
   public Image getImage(String prompt) {
+    HashMap<String, String> getImageDetails = new HashMap<>();
+    getImageDetails.put("prompt", prompt);
```
Comment (sanchitmalhotra126) [inline org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java:17]: Can you add `"prompt"` to `ClientMessageDetailKeys` and reference here?

**Review by sanchitmalhotra126 — COMMENTED** [review]
Couple small things, otherwise looks good!

File: org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java (hunk 2 above)
Comment (bencodeorg) [reply org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java:13]: That does make things a lot cleaner, thanks

**Pull request merged**